src/index.tsx

```diff
@@ -0,0 +1,5 @@
+import {extendTheme} from './core/extendTheme'
+import BaseProvider from './core/BaseProvider'
+import Hello from './Hello'
+
+export {extendTheme, BaseProvider, Hello}
```

Comment:
`Hello` ?

Reply:
Hello component is an example for theme

Reply:
**WelcomeScreen** should be more make sense